Rule-011.postman_collection.json

```diff
@@ -0,0 +1,9585 @@
+              "options": {
+                "raw": {
+                  "language": "json"
+                }
+              }
+            },
+            "url": {
+              "raw": "http://gateway.openfaas:8080/function/off-transaction-monitoring-service-rel-1-0-0/quoteReply",
+              "protocol": "http",
+              "host": ["frmfaas", "sybrin", "com"],
```

Comment:
I'm not sure why your host has a bunch of "host": ["frmfaas", "sybrin", "com"]. and the complete URL is gateway.openfaas please can you fix it?

Reply:
Fixed boss